docs/Guide/utilities/stopwatch.mdx

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 2
+title: Using StopWatch
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Provides a set of methods and properties that you can use to accurately measure elapsed time.
```

Comment:
This can be a bit more descriptive. I know, I know, you copied it over. That's bad on us too.